osidb/api_views.py

```diff
@@ -166,7 +168,13 @@ def get_valid_http_methods(cls: ViewSet, excluded: list[str] = None) -> list[str
         "or the CVE number corresponding to a Flaw"
     ),
 )
-
+jira_token_param = OpenApiParameter(
+    name="JiraAuthentication",
```

Comment:
I am a bit late but I just noticed that for Bugzilla we use naming `Bugzilla-Api-Key` for the header and it would be nice to unify it. `Jira-Api-Key` would also make more sense to me, it is more clear what it is supposed to be.

Reply:
I think neither of those are suitable names. In Bugzilla it is actually called API key, however in Jira it is called Personal Access Tokens and thus I would be for `Jira-Access-Token`.

Reply:
I think either option would be good, one follows the current pattern and other uses jira naming, I have submitted as `Jira-Api-Key` but if we prefer I can change to `Jira-Access-Token`.